src/service/template-impl.js

```diff
@@ -319,7 +333,7 @@ export class Templates {
     } else if (opt_querySelector) {
       return scopedQuerySelector(parent, opt_querySelector);
     } else {
-      return parent.querySelector('template, script[type="text/plain"]');
+      return parent.querySelector('template[type], script[type="text/plain"]');
```

Comment:
There's an error message in `findTemplate` that should be updated to reflect that it's also looking for `type` attr: `Template must be defined in a <template> or `

Reply:
I think as a user error it's ok. It's not meant to be the exact error - more of a "look into docs" sort of thing. I think even one of the teams is reworking all errors to just point straight at docs.